src/components/Glp/SwapErrorModal.tsx

```diff
@@ -0,0 +1,75 @@
+import { BigNumber } from "@ethersproject/bignumber";
+import { useEffect, useState } from "react";
+import { ethers } from "ethers";
+import { getNativeToken } from "../../config/Tokens";
+import { InfoToken, Token } from "../../domain/tokens/types";
+import Modal from "../Modal/Modal";
+import { get1InchSwapUrl } from "../../domain/common";
+
+const { AddressZero } = ethers.constants;
+
+type Props = {
+  swapToken: Token;
+  isVisible: boolean;
+  setIsVisible: () => void;
+  chainId: number;
+  glpAmount: BigNumber;
+  usdgSupply: BigNumber;
+  totalTokenWeights: BigNumber;
+  glpPrice: BigNumber;
+  infoTokens: {
+    [key: string]: InfoToken;
+  };
+};
+
+interface LowestFeeToken extends Token {
+  fees: number;
+}
+
+export default function SwapErrorModal({
+  swapToken,
+  isVisible,
+  setIsVisible,
+  chainId,
+  glpAmount,
+  usdgSupply,
+  totalTokenWeights,
+  glpPrice,
+  infoTokens,
+}: Props) {
+  const [lowestFeeToken, setLowestFeeToken] = useState<LowestFeeToken>();
+  useEffect(() => {
+    setLowestFeeToken(
+      getLowestFeeTokenForBuyGlp(
+        chainId,
+        glpAmount,
+        glpPrice,
+        usdgSupply,
+        totalTokenWeights,
+        infoTokens,
+        swapToken?.address
```

Comment:
here we check if swapToken is defined, however on line 53 it's just `swapToken.address`

i think better:
- if swapToken can be undefined then it should be handled in dependency array as well (and i guess in Props?)
- if it can no be undefined – then just remove `?` to make it consistent

Reply:
Done!